cmd/acr/purge.go

```diff
@@ -240,6 +245,7 @@ func getTagsToDelete(ctx context.Context,
 	acrClient api.AcrCLIClientInterface,
 	repoName string,
 	filter *regexp.Regexp,
+	excludeFilter []string,
```

Comment:
You should pass a `*regexp.Regexp` instead of `[]string` for `excludeFilter`. The reason is explained in the next comment.